build.gradle

```diff
@@ -79,19 +147,27 @@ configure(subprojects.findAll { it.name.startsWith("maven-") }) {
     }
 }
 
-subprojects {
-    buildscript {
-        repositories {
-            google()
-            jcenter()
-            maven { url 'https://dl.bintray.com/kotlin/ktor' }
-            maven { url 'https://dl.bintray.com/kotlin/kotlin-eap' }
-        }
-        dependencies {
-            classpath "com.android.tools.build:gradle:3.4.1"
+
+if (build_snapshot_train) {
+    println "Hacking test tasks, removing stress and flaky tests"
+    allprojects {
+        tasks.withType(Test).all {
+            // Add: exclude '**/*TestName*' here
         }
     }
-    repositories {
-        maven { url 'https://dl.bintray.com/kotlin/kotlin-eap' }
+
+    println "Manifest of kotlin-compiler-embeddable.jar for coroutines"
+    configure(subprojects.findAll { it.name == "kotlinx-coroutines-core" }) {
+        configurations.matching { it.name == "kotlinCompilerClasspath" }.all {
+            resolvedConfiguration.getFiles().findAll { it.name.contains("kotlin-compiler-embeddable") }.each {
+                def manifest = zipTree(it).matching {
+                    include 'META-INF/MANIFEST.MF'
+                }.getFiles().first()
+
+                manifest.readLines().each {
+                    println it
```

Comment:
debug?

Reply:
Yep, for train purposes. We have the same line in ktor.